### ISSUE TITLE
use model to run_CIL.py

### ISSUE THREAD

**Issue by ll2255-git** [mvpcom/carlaILTrainer#6]
hi,I use train.py to store a model.ckpt-30.data-00000-of-00001,just running 30 steps to test  whether model can test carla-server,but there has some error imformations when I had run command "python run_CIL.py"
NotFoundError (see above for traceback): Key Network/Branch_0/Variable not found in checkpoint
	 [[Node: save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]

**Comment by merryHunter** [comment]
Hi @243225530 , Basically, I didn't test that model in carla server, but I believe in the latest version I've done I wasn't able to handle properly speed branch, that's why you need to adapt code in run_CIL in order to have 4 branches, not 5. Hope that helps.

**Comment by ll2255-git** [comment]
@merryHunter thanks for your answer, and what should I do if I want to test the model in carla, now I had get the files such as model.ckpt.data-00000-of-00001 and so on  by train.py, and put its in model folder, finally, using "python run_CIL.py" command. But It is wrong, and what should I do if I want to test the model in carla. Thank you very much!

**Comment by melodyhappy** [comment]
Have you solved this problem? i delete the speed branch,but the error still remains.

**Comment by ll2255-git** [comment]
Hi, @melodyhappy I have solved this problem, the programs can run normally, this project has a error at training code segments, under the comments "# every 50000 step, multiply learning rate by half", codes "netTensors['output']['branchesOutputs'][i]" should change to "netTensors['output']['output'][i]", this error found the controlNet function return value structs, And you should also change the imitation_learning.py,could you tell me a mail or other contact imformations, I will send the "imitation_learning.py" file to you!

**Comment by melodyhappy** [comment]
wow, thx a lot!!!!!!
My email is 548985709@qq.com.

**Comment by ll2255-git** [comment]
@melodyhappy ok, I will send to you after 2.00 pm

**Comment by 243225530** [comment]
@melodyhappy ok, I will send to you after 2.00 pm

**Comment by mvpcom** [comment]
@243225530 You can request a pull if you think your changes are helpful for the others.

**Comment by m273033** [comment]
@243225530 @melodyhappy  Can you please share the imitation_learning.py somewhere?

**Comment by sayanmandal** [comment]
@243225530  Can you please send me the updated code to me. I am facing the same problem. My email is: sayan007roni@gmail.com

**Comment by Warsin** [comment]
@243225530 Can you please send me the updated code to me. I am facing the same problem. My email is: 939250412@qq.com

**Comment by chenxuejie** [comment]
@243225530 Can you please send me the updated code to me. I am facing the same problem. My email is: xuejiechen52@gmail.com

**Comment by se-yi** [comment]
@243225530  Could you please send me the updated "imitation_learning.py" as well ? My email is: seungeun.yi10@gmail.com

**Comment by fenjiro** [comment]
could you please send me the updated "imitation_learning.py" as well. My email is: 
fenjiro@gmail.com

**Comment by kimna4** [comment]
@243225530  Could you please send me the updated "imitation_learning.py" as well ? My email is:  
kiminhan0527@gmail.com

Thank you in advance.

**Comment by chowkamlee81** [comment]
@243225530 , please send me the updated "imitation_learning.py" as well ? My email is: chowkamlee81@gmail.com.

Kindly help

**Comment by chowkamlee81** [comment]
@fenjiro @kimna4 @se-yi @chenxuejie , please send me the updated "imitation_learning.py" as well ? My email is: chowkamlee81@gmail.com.

**Comment by Deepak3994** [comment]
@mvpcom can you kindly update the code in imitaionlearning.py. Do the needful

**Comment by PankajJain5889** [comment]
can you send the changes to pankaj@colostate.edu
